.github/workflows/test.yml

```
@@ -14,27 +14,29 @@ jobs:
           pre-commit install
           pre-commit run --all-files
 
-
   tests:
     name: Run Tests
     runs-on: ubuntu-latest
 
     services:
+      web:
+        # other configuration...
+        env_file:
```

actionlint (1.7.4)

unexpected key "env_file" for "services" section. expected one of "credentials", "env", "image", "options", "ports", "volumes"

(syntax-check)